### PR TITLE
Issue 1573 - Fix Links to generalist repositories in Commons

### PR DESCRIPTION
**Pull request by jrhoads**
## Purpose
Fix links to generalist repositories in Commons

closes: https://github.com/datacite/datacite/issues/1573 https://github.com/datacite/datacite/issues/1578

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

- [ ] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Reviewer, please remember our [guidelines](https://datacite.atlassian.net/wiki/spaces/TEC/pages/1168375809/Pull+Request+Guidelines):

- Be humble in the language and feedback you give, ask don't tell.
- Consider using positive language as opposed to neutral when offering feedback. This is to avoid the negative bias that can occur with neutral language appearing negative.
- Offer suggestions on how to improve code e.g. simplification or expanding clarity.
- Ensure you give reasons for the changes you are proposing.

### CODE REVIEW

File: src/components/RepositoryDetail/RepositoryDetail.tsx [hunk 1]
```diff
@@ -127,7 +128,7 @@ export const RepositoryHeadInfo: React.FunctionComponent<Props> = ({
           <meta name="og:title" content={info.title} />
           <meta name="og:url" content={info.pageUrl} />
           <meta name="og:image" content={info.imageUrl} />
-          <meta name="og:type" content="organization" />
+          <meta name="og:type" content="repository" />
```
Comment (digitaldogsbody) [inline src/components/RepositoryDetail/RepositoryDetail.tsx:131]: I am curious about this. Neither `organization` or `repository` is in the OGP types list: https://ogp.me/#types. Do we need to define a namespace that has them and prefix with that?

**Review by digitaldogsbody — APPROVED** [review]
Looks good to me

File: src/components/RepositoryDetail/RepositoryDetail.tsx (hunk 1 above)
Comment (jrhoads) [reply src/components/RepositoryDetail/RepositoryDetail.tsx:131]: Originally this had been copied from the organization page
https://github.com/datacite/akita/blob/8f5d72bf9a3b531dde34efff70b9e58f5df0107f/src/pages/ror.org/%5Brorid%5D.tsx#L280

I agree.  Neither organization, nor repository are part of the OPG vocabulary.

Comment (jrhoads) [reply src/components/RepositoryDetail/RepositoryDetail.tsx:131]: I think this is worth further investigation but not necessarily a blocker for this work.

Comment (digitaldogsbody) [reply src/components/RepositoryDetail/RepositoryDetail.tsx:131]: Agreed :)

**Pull request merged**